models.py

```diff
@@ -47,4 +49,39 @@ def serialize(self):
             'recommendation_text': self.recommendation_text,
             'active': self.active,
         }
+        
+class Score(db.Model):
+    __tablename__ = 'scores'
 
+    id = db.Column(db.Integer, primary_key=True)
+    criterion_id = db.Column(db.Integer, db.ForeignKey("criteria.id"), nullable=False)
+    state = db.Column(db.String(), nullable=False)
+    meets_criterion = db.Column(db.Boolean())
+    created_at = db.Column(db.DateTime)
+
+    def __init__(self, criterion_id, state, meets_criterion):
+        self.criterion_id = criterion_id
+        self.state = state
+        self.meets_criterion = meets_criterion
+        self.created_at = datetime.datetime.utcnow()
+
+    def __repr__(self):
+        return '<id {}>'.format(self.id)
+
+    @validates('state')
+    def validate_state(self, key, value):
+        assert value in ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", 
+             "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", 
+             "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]
+        return value
```

Comment:
How come you return `value` here? Is this something I should be doing when validating input?

Reply:
So it looks like the validator methods in SQLAlchemy modify the column being validated based on the return statement. If you take out this return and run the test, the "state" column always gets set to None and the tests fail!

This is mentioned briefly in the [SQLAlchemy docs](https://docs.sqlalchemy.org/en/13/orm/mapped_attributes.html) and their examples have a return statement.